content/en/blog/_posts/2021-04-15-local-storage-features-go-beta.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: blog
+title: 'Local Storage: Storage Capacity Tracking, Distributed Provisioning and Generic Ephemeral Volumes hit Beta
+date: 2021-04-15
+slug: 2021-04-15-local-storage-features-go-beta
+---
+
+ **Authors:** Patrick Ohly (Intel)
+
+Several features are getting promoted to beta in Kubernetes 1.21 and
+the associated CSI external-provisioner release that simplify the
+development of Container Storage Interface (CSI) drivers which manage
+storage locally on a node.
+
+This blog post explains how such drivers worked before and how these
+features help make drivers simpler.
+
+## The good...
+
+There are drivers for local storage, like TopoLVM for traditional
+disks and PMEM-CSI for persistent memory. They work and are ready for
+usage today also on older Kubernetes release, but making that possible
+was not trivial.
+
+The first problem is volume provisioning: it is handled through the
+Kubernetes control plane. Some central component must react to
+persistent volume claims (PVC) and create volumes. Usually, that is
```

Comment:
If you want to, you can make persistent volume claims be a link to https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims